app/validators.py

```diff
@@ -26,3 +26,32 @@ def validate(cls, nip: str) -> str:
         if checksum_calculated != int(nip[9]):
             raise ValueError("Nip failed checksum validation")
         return cls(nip)
+
+class RegonNumber(str):
+    @classmethod
+    def __get_validators(cls):
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, regon: str) -> str:
```

Comment:
Could use some unit test to make sure regon validation works properly

Reply:
1. Correct 9-digit number
2. Wrong 9-digit number
3. Correct 14-digit number
4. Wrong 14-digit number